### ISSUE TITLE
conflict with firegento/magesetup2

### ISSUE THREAD

**Issue by kkrieger85** [Magenerds/GermanLaw#15]
Little conflict with firegento/magesetup2

doubles text after price by plugin "magenerds_germanlaw_priceRender" if both extensions are installed

**Comment by sydekumf** [comment]
Hi there,

I am not sure, what exactly is the problem. There was an issue with double display of the tax information. But this was already fixed. Do you use the latest release of this extension?

**Comment by kkrieger85** [comment]
Hi, 
I used:
```
"name": "magenerds/germanlaw",
            "version": "1.0.10",
            "source": {
                "type": "git",
                "url": "https://github.com/Magenerds/GermanLaw.git",
                "reference": "88679f4d0501bdbf9f7f914e951a142ae2a28cf7"
            },
```
and 

```
"name": "firegento/magesetup2",
            "version": "dev-develop",
            "source": {
                "type": "git",
                "url": "https://github.com/firegento/firegento-magesetup2.git",
                "reference": "a89faae69c803218a337b701d5697c5cb89d4c57"
            },
```

Problem was double display of tax information as you guessed.

**Comment by sydekumf** [comment]
"Problem was"? Is it fixed with 1.0.10 of german law?

**Comment by sydekumf** [comment]
I just looked into magesetup2 and it seems they use the same plugin for after price rendering, so this is expected behavior what you describe. You should decide which module you want to use, if you use both, of course it will be displayed two times ;-)
